package-lock.json

```diff
@@ -1,19 +1,14307 @@
 {
   "name": "regl",
   "version": "2.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Let's not touch the version of package-lock in this PR.

Reply:
This has been reverted now too. I just undid the last 2 commits.